habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -198,13 +214,98 @@ class TeleportActionConfig(ActionConfig):
 @attr.s(auto_attribs=True, slots=True)
 class VelocityControlActionConfig(ActionConfig):
     type: str = "VelocityAction"
-    # meters/sec
-    lin_vel_range: List[float] = [0.0, 0.25]
-    # deg/sec
-    ang_vel_range: List[float] = [-10.0, 10.0]
+    lin_vel_range: List[float] = [0.0, 0.3]  # meters/sec
+    ang_vel_range: List[float] = [-0.45, 0.45]  # rad/sec
+    ang_vel_range_camera_pitch: List[float] = [-0.45, 0.45]  # rad/sec
+    ang_range_camera_pitch: List[float] = [-1.57, 0.43]  # rad
+    time_step: float = 0.1  # seconds
+    enable_scale_convert: bool = False
```

Comment:
Making this True by default. Also, this cannot be changed by the participants during the challenge.
```suggestion
    enable_scale_convert: bool = True
```